css/main.css

```diff
@@ -1,207 +1,194 @@
+	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("https://bit.ly/3dletUn");
+	height: 400px;
+	background-position: center;
+	background-repeat: no-repeat;
+	background-size: cover;
+	position: relative;
 }
 
 .ingredients-text {
-    text-align: center;
-    position: relative;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
-    color: white;
+	text-align: center;
+	position: relative;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	color: white;
 }
 
 #wingplatter {
-    background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("../images/wingplatter.jpg");
-    height: 400px;
-    background-position: left;
-    background-repeat: repeat;
-    position: relative;
-    width: 100%;
-    background-size: 100%;
+	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("../images/wingplatter.jpg");
+	height: 400px;
+	background-position: left;
+	background-repeat: repeat;
+	position: relative;
+	width: 100%;
+	background-size: 100%;
 }
 
 .table {
-    margin-right: auto;
-    margin-left: auto;
-    width: 200px;
+	margin-right: auto;
+	margin-left: auto;
+	width: 200px;
 }
 
 #hours {
-    text-align: center;
-    font-family: 'Times New Roman', Times, serif;
+	text-align: center;
 }
 
 /*for mainpage.html wingassortment text*/
 
 .connectTexts {
-    text-align: left;
-    padding: 40px;
+	text-align: left;
+	padding: 40px;
 }
 
 .row {
-    display: flex;
-    background-color: lightgray;
+	display: flex;
+	background-color: lightgray;
 }
 
 #connect img {
-    padding: 20px;
-    object-fit: contain;
-    width: 500px;
+	padding: 20px;
+	object-fit: contain;
+	width: 500px;
 }
 
 .column {
-    float: left;
-    width: 50%;
+	float: left;
+	width: 50%;
 }
 
 .column img {
-    width: 400px;
-    float: right;
-    padding: 20px;
-    width: 600px;
-    object-fit: contain;
+	width: 400px;
+	float: right;
+	padding: 20px;
+	width: 600px;
+	object-fit: contain;
 }
 
 /* css for starbull.php*/
 
 .cwing {
-    background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("../images/cwing.jpg");
-    height: 400px;
-    background-position: left;
-    background-repeat: repeat;
-    position: relative;
-    width: 100%;
-    background-size: 100%;
+	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("../images/cwing.jpg");
+	height: 400px;
+	background-position: left;
+	background-repeat: repeat;
+	position: relative;
+	width: 100%;
+	background-size: 100%;
 }
 
 /*css for us.html*/
 
 p {
-    font-size: 25px;
-    font-family: Georgia, 'Times New Roman', Times, serif;
+	font-size: 25px;
 }
 
 h3 {
-    font-size: 35px;
-    font-family: Georgia, 'Times New Roman', Times, serif;
+	font-size: 35px;
 }
 
 /* Sticky elements for Menu page */
 
 /* ?? wondering if this media query is part of the answer for adjusting accordion header sticky height after hamburger menu is toggled ?? 
 
```

Comment:
Should we delete this? I don't this it's relevant anymore. @margeschrec correct me if I'm wrong.

Reply:
Yes, that can be deleted